.github/workflows/test_action.yaml

```diff
@@ -0,0 +1,23 @@
+name: GitHub Actions Demo
```

Comment:
Do we need this?

Reply:
No, that was for my testing, and I forgot about it. Removed now.